### PR TITLE
fix(packages): change to valid patternplate version

### PR DESCRIPTION
**Pull request by LKuechler**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,8 +88,8 @@
     "@commitlint/config-conventional": "^6.0.2",
     "@commitlint/prompt-cli": "^6.0.2",
     "@commitlint/travis-cli": "^6.0.2",
-    "@patternplate/cli": "2",
-    "@patternplate/render-styled-components": "2",
+    "@patternplate/cli": "2.0.0-93",
```
Comment (marionebl) [inline package.json]: We are beyond the prereleaseas now, the most recent versions are `2.0.x`

**Comment by Jumace** [comment]
@marionebl could you change the commit message? Not everyone knows what 'pp' means and there are still enough characters left to write the whole name ;)

**Comment by marionebl** [comment]
@Jumace I assume you'll squash the commits in here when merging anyway, so please reword the commit message then. :)

**Comment by LKuechler** [comment]
@marionebl currently it is just your commit but i can change this later :)

**Pull request merged**